sstables/mx/writer.cc

```diff
@@ -1259,7 +1259,7 @@ void writer::write_clustered(const clustering_row& clustered_row, uint64_t prev_
         }
     }
 
-    const bool is_dead = !clustered_row.is_live(_schema, clustered_row.tomb().regular(), gc_clock::now());
+    const bool is_dead = !clustered_row.is_live(_schema, tombstone(), gc_clock::now());
```

Comment:
Please fold this into the commit which introduced this line, AFAIK said commit is also introduced in this PR.

Reply:
I have folded the coommits.